llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -5897,6 +5898,7 @@ bool SPIRVToLLVM::translate(ExecutionModel entryExecModel, const char *entryName
   if (!m_entryTarget)
     return false;
 
+  m_shaderStage = convertToStageShage(entryExecModel);
```

Comment:
Should this function be named to `convertToShaderStage`? Maybe change it in another PR.